src/raiden_libs/cli.py

```diff
@@ -207,3 +209,11 @@ def connect_to_blockchain(
     hex_addresses = {key: to_checksum_address(value) for key, value in addresses.items()}
     log.info("Contract information", addresses=hex_addresses, start_block=start_block)
     return web3, contracts, start_block
+
+
+def maybe_setup_sentry(enable_flask_integration: bool = False) -> None:
+    sentry_dsn = os.environ.get("SENTRY_DSN")
+    if sentry_dsn is not None:
+        sentry_sdk.init(
+            dsn=sentry_dsn, integrations=[FlaskIntegration()] if enable_flask_integration else []
```

Comment:
While you're at it, you could also add `version=pkg_resources.get_distribution("raiden-services").version`

Reply:
Do you mean release? https://docs.sentry.io/error-reporting/configuration/?platform=python#release

Reply:
Yes, I do!